### PR TITLE
#59522: adjust messages

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Adjusts messages from #59522 (see discussion in #65014).

CC @coreaut

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
pulp_repo

### CODE REVIEW

**Review by coreaut — CHANGES_REQUESTED** [review]
One pulp.io got overlooked :)

File: changelogs/fragments/59522-renamed-module-tls-client-auth-params-to-avoid-overlaping-with-fetch_url.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 bugfixes:
-  - pulp_repo - the ``client_cert`` and ``client_key`` options were used for both requests to pulp.io and for the repo to sync with, resulting in errors when they were used. Use the new options ``feed_client_cert`` and ``feed_client_key`` for client certificates that should only be used for repo synchronisation, and not for communication with pulp.io. (https://github.com/ansible/ansible/issues/59513)
+  - pulp_repo - the ``client_cert`` and ``client_key`` options were used for both requests to pulp.io and for the repo to sync with, resulting in errors when they were used. Use the new options ``feed_client_cert`` and ``feed_client_key`` for client certificates that should only be used for repo synchronisation, and not for communication with the Pulp instance. (https://github.com/ansible/ansible/issues/59513)
```
Comment (coreaut) [inline changelogs/fragments/59522-renamed-module-tls-client-auth-params-to-avoid-overlaping-with-fetch_url.yaml]:
```suggestion
  - pulp_repo - the ``client_cert`` and ``client_key`` options were used for both requests to pulp Instance and for the repo to sync with, resulting in errors when they were used. Use the new options ``feed_client_cert`` and ``feed_client_key`` for client certificates that should only be used for repo synchronisation, and not for communication with the Pulp instance. (https://github.com/ansible/ansible/issues/59513)
```

Comment (felixfontein) [reply changelogs/fragments/59522-renamed-module-tls-client-auth-params-to-avoid-overlaping-with-fetch_url.yaml]: Thanks, fixed it!

**Review by coreaut — APPROVED** [review]

**Comment by coreaut** [comment]
shipit :tada:

**Comment by felixfontein** [comment]
@coreaut thanks for reviewing! :)

**Pull request merged**